cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/testevents/TestEventOperator.java

```diff
@@ -34,6 +32,7 @@
 import static com.exactpro.cradle.cassandra.dao.testevents.TestEventEntity.*;
 
 @Dao
+@DefaultNullSavingStrategy(NullSavingStrategy.DO_NOT_SET)
```

Comment:
note: If I understand the documentation correctly this is the default strategy if nothing is specified explicitly.
> If you don’t define a DAO-level default, any method that does not declare its own value defaults to [DO_NOT_SET](https://docs.datastax.com/en/drivers/java/4.17/com/datastax/oss/driver/api/mapper/entity/saving/NullSavingStrategy.html#DO_NOT_SET)

https://docs.datastax.com/en/developer/java-driver/4.17/manual/mapper/daos/null_saving/index.html

Do we need this annotation here at all?